domains/efbotlink.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Heroku403",
+        "email": "juhisharma0757@gmail.com"
+    },
+    "record": {
+        "CNAME": "amorphous-tiger-uqyq9wgl8mh7keh8qonqzxst.herokudns.com"
```

Comment:
Your site is empty, heroku says "There's nothing here, yet."

Reply:
Yeah becuz I turn off the dyno for future is it a problem so I can switch it back ??